### PR TITLE
Improvements for arbiter diagnostic work

### PR DESCRIPTION
**Pull request by slacAWallace**
Changes made to facilitate the PMPS diagnostics

### CODE REVIEW

File: lcls-twincat-pmps/Library/GVLs/PMPS_GVL.TcGVL [hunk 1]
```diff
@@ -3,37 +3,60 @@
   <GVL Name="PMPS_GVL" Id="{b2e0edd3-11b0-047f-0282-aaa0e3c1afb7}">
     <Declaration><![CDATA[{attribute 'qualified_only'}
 VAR_GLOBAL
-    stReqFBDBeamParamsSum    :    ST_BeamParams;
-    stReqMethBeamParamSum    :    ST_BeamParams;
-    stCurrentBeamParameters    :    ST_BeamParams;
+    {attribute 'pytmc' := '
+        pv: @(PREFIX)RequestedBP
+        archive: 1Hz monitor
```
Comment (klauer) [inline lcls-twincat-pmps/Library/GVLs/PMPS_GVL.TcGVL:8]: This is the default setting, I believe, so it's not necessary here.

Comment (klauer) [reply lcls-twincat-pmps/Library/GVLs/PMPS_GVL.TcGVL:8]: Nope, `scan` is the default. Why'd you feel it was necessary to change here?

Comment (slacAWallace) [reply lcls-twincat-pmps/Library/GVLs/PMPS_GVL.TcGVL:8]: hmm, I must be misunderstanding something... I expect this to change, but don't care about recording the changes at anything >1Hz, and I also expect it to be possible for many of these things to remain static for longer periods of time, hence the monitor setting.

Comment (klauer) [reply lcls-twincat-pmps/Library/GVLs/PMPS_GVL.TcGVL:8]: Yeah, `scan` and `monitor` are still confusing to me as far as the archiver configuration goes. You should be safe with the default in this scenario regardless, as we record all pragma'd things at 1Hz.

**Pull request merged**